src/libraries/System.Runtime.InteropServices/src/System/Runtime/InteropServices/PosixSignalContext.cs

```diff
@@ -0,0 +1,40 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System.Runtime.InteropServices
+{
+    /// <summary>
+    /// Provides data for a <see cref="PosixSignalRegistration"/> event.
+    /// </summary>
+    public sealed class PosixSignalContext
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="PosixSignalContext"/> class.
+        /// </summary>
+        public PosixSignalContext(PosixSignal signal)
+        {
+            Signal = signal;
+        }
```

Comment:
Does this ctor exist just for testability?  I don't see it being used in the implementation.

Can you add a simple test for it?